istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -87,7 +87,6 @@ data:
     disableMixerHttpReports: false
     {{- end }}
 
-
```

Comment:
Missing `disableMixerHttpReports`?

Reply:
it's already there somehow, I think a bad merge occured

Reply:
ah ok